databricks/koalas/tests/test_dataframe.py

```diff
@@ -16,6 +16,7 @@
 from datetime import datetime
 from distutils.version import LooseVersion
 import inspect
+import re
```

Comment:
nit: Maybe I don't think we don't necessarily need `re` module because there is no string have to escape ??

```python
ks_err_msg = re.escape("multiplication can not be applied to a string literal")
ks_err_msg = "multiplication can not be applied to a string literal"  # This also could be fine.
```

Reply:
Sounds good! Removed  `re` module usage.